libs/fetcher.js

```diff
@@ -223,7 +224,7 @@ function executeRequest (request, resolve, reject) {
     var service;
     try {
         service = Fetcher.getService(request.resource);
-        if (!service[op]) {
+        if ([OP_CREATE,OP_READ,OP_UPDATE,OP_DELETE].indexOf(op) < 0 || !service[op]) {
```

Comment:
@nysd Thanks for your contribution!

How about moving this check before the line above, and checking using `===` explictly to avoid allocating an array and doing an indexOf every time request is executed?